docker/Dockerfile

```diff
@@ -0,0 +1,94 @@
+# Base image for the container - using Eclipse Temurin with OpenJDK 17 and Jammy base
+# This is the same image used in the offical Spark dockerfile which was used as reference
+# for this dockerfile
+FROM eclipse-temurin:17-jdk-jammy
```

Comment:
I think UC is JVM 11.  Could be mistaken.

Reply:
We are compiling the UC upon the first run. So any version above 11 should work based on the sbt code in the screenshot below.
I'm happy though to change the version to 11 if it makes more sense.
I'll take your steer on this.

![image](https://github.com/unitycatalog/unitycatalog/assets/1816932/9b8373fe-c148-462a-a377-3374324d703b)